test/jest/__mock__/stripesSmartComponents.mock.js

```diff
@@ -4,6 +4,22 @@ jest.mock('@folio/stripes/smart-components', () => ({
   ClipCopy: jest.fn(() => null),
   makeQueryFunction: jest.fn((value) => value),
   CheckboxFilter: jest.fn(() => null),
+  NoteCreatePage: (props) => {
+    return (
+      <div>
+        <div>NoteCreatePage</div>
+        {props.renderReferredRecord()}
+      </div>
+    );
+  },
```

Comment:
I do not see usage of `NoteCreatePage` inside `NoteEditRoute`. Could you remove it?

Reply:
Updated file